### PR TITLE
pdb doc - workaround for double semicolon in strings

### PR DESCRIPTION
**Pull request by gotcha**
give a hint

see https://github.com/gotcha/ipdb/issues/172

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@gotcha

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Comment by gotcha** [comment]
I have signed the Contributor agreement and updated my github username in bugs.python.org.

File: Doc/library/pdb.rst [hunk 1]
```diff
@@ -233,7 +233,8 @@ Multiple commands may be entered on a single line, separated by ``;;``.  (A
 single ``;`` is not used as it is the separator for multiple commands in a line
 that is passed to the Python parser.)  No intelligence is applied to separating
 the commands; the input is split at the first ``;;`` pair, even if it is in the
-middle of a quoted string.
+middle of a quoted string. To be able to use a string with double semi-colon,
+use the following workaround: ``';'';'`` or ``";"";"``.
```
Comment (JelleZijlstra) [inline Doc/library/pdb.rst]:
```suggestion
middle of a quoted string. A workaround for strings with double semicolons
is to use implicit string concatenation ``';'';'`` or ``";"";"``.
```

**Review by JelleZijlstra — APPROVED** [review]

**Comment by gotcha** [comment]
@JelleZijlstra Thanks for merging !

**Pull request merged**